src/main/kotlin/com/software/modsen/carservice/dto/request/CarRequest.kt

```diff
@@ -0,0 +1,8 @@
+package com.software.modsen.carservice.dto.request
+
+data class CarRequest(
+        var model: String? = null,
+        var number: String? = null,
+        var color: String? = null
```

Comment:
If request, do not make fields nullable if they don't

Reply:
Changed